packages/gatsby-theme-docs/utils/process-table-of-content-fields.spec.js

```diff
@@ -85,7 +85,7 @@ describe('processTableOfContentFields', () => {
           title: 'In a title',
           items: [
             {
-              url: '#anchor-in-title-links-here',
+              url: '#anchor-in-title-links-here-',
```

Comment:
Adjustment for `github-slugger 1.4.0`
See more information [here](https://github.com/Flet/github-slugger/pull/38)